pkg/karmadactl/get.go

```diff
@@ -345,6 +345,20 @@ func (g *CommandGetOptions) checkPrintWithNamespace(mapping *meta.RESTMapping) b
 func (g *CommandGetOptions) getObjInfo(wg *sync.WaitGroup, mux *sync.Mutex, f cmdutil.Factory,
 	cluster string, objs *[]Obj, allErrs *[]error, args []string) {
 	defer wg.Done()
+
+	restClient, err := f.RESTClient()
+	if err != nil {
+		*allErrs = append(*allErrs, err)
+		return
+	}
+
+	// check if it is authorized to proxy this member cluster
+	request := restClient.Get().RequestURI(fmt.Sprintf(proxyURL, cluster) + "apis")
```

Comment:
Use `api` will be better, because it's response is smaller than `apis`.

Reply:
done